tests/map.config.test.json

```diff
@@ -0,0 +1,316 @@
+{
```

Comment:
We need to pass the config on the constructor.
Because on MapComponent.spec.ts test we instantiate the component MapComponent. And on the mount function of MapComponent we call the function initializeMap on RennesApp. In initializeMap the context is set on map. And if we don't pass an empty context by argument, the global context was set and we have an error related to WebGL.